ovos_utils/log.py

```diff
@@ -37,65 +29,99 @@ class LOG:
         >>> LOG('custom_name').debug('Another message')
         13:13:10.462 - custom_name - DEBUG - Another message
     """
-
-    _custom_name = None
-    handler = None
-    level = None
-
-    # Copy actual logging methods from logging.Logger
-    # Usage: LOG.debug(message)
-    debug = _make_log_method(logging.Logger.debug)
-    info = _make_log_method(logging.Logger.info)
-    warning = _make_log_method(logging.Logger.warning)
-    error = _make_log_method(logging.Logger.error)
-    exception = _make_log_method(logging.Logger.exception)
+    base_path = "stdout"
+    fmt = '%(asctime)s.%(msecs)03d - ' \
+          '%(name)s - %(levelname)s - %(message)s'
+    datefmt = '%Y-%m-%d %H:%M:%S'
+    formatter = logging.Formatter(fmt, datefmt)
+    max_bytes = 50000000
```

Comment:
Is this big? This seems kinda big.

Reply:
50MB is pretty standard for log size limit